src/utils/delegations/signWithdrawalTx.ts

```diff
@@ -132,10 +132,12 @@ export const signWithdrawalTx = async (
     withdrawPsbtTxResult.fee,
   );
 
-  emitBroadcastEvent();
+  // TODO: uncomment back
+  // emitBroadcastEvent();
 
   // Broadcast withdrawal transaction
-  await pushTx(withdrawalTxHex);
+  // await pushTx(withdrawalTxHex);
+
```

Comment:
Can you revert these? Or we would want to test more?

Reply:
that's just for testing, yes we need to revert it after Benedikt can test with his wallet